libraries/pbrlib/genglsl/lib/mx_environment_fis.glsl

```diff
@@ -34,7 +34,7 @@ vec3 mx_environment_radiance(vec3 N, vec3 V, vec3 X, vec2 alpha, int distributio
 
         // Compute the half vector and incoming light direction.
         vec3 H = mx_ggx_importance_sample_NDF(Xi, alpha);
-        vec3 L = -reflect(V, H);
+        vec3 L = fd.refraction ? mx_refraction_solid_sphere(-V, H, fd.ior.x) : -reflect(V, H);
```

Comment:
Just a note for the future that we need to split this switch in two later to support thin-walled materials. We then need to have separate control of transmission and refraction on/off, since we can have transmission without refraction in that case, but still transmission with a roughness.